webapp/app/[locale]/get-started/_components/fundWallet.tsx

```diff
@@ -45,7 +45,7 @@ const FundMethod = function ({
       href={url}
       onClick={addTracking()}
     >
-      {icon}
+      <div className="w-7">{icon}</div>
```

Comment:
Here I think the icons would better work with `w-5.5`. Can you enable it in `tailwind.config.ts` and use that? (I'm using that value seeing the largest `width` of each icon, but I haven't tested it)